cms/templates/videos_index.html

```diff
@@ -60,13 +60,15 @@ <h3 class="sr">${_("Page Actions")}</h3>
-                <p>${_("For a video to play on different devices, it needs to be available in multiple formats. After you upload an original video file in .mp4 or .mov format on this page, an automated process creates those additional formats and stores them for you.")}</p>
+                <p>${_("For a video to play on different devices, it needs to be available in multiple formats. After you upload an original video file in .mp4 or .mov format on this page, an automated process creates those additional formats and stores them on external video host sites for you.")}</p>
                 <h3 class="title-3">${_("Monitoring files as they upload")}</h3>
-                <p>${_("Each video file that you upload needs to reach the video processing servers successfully before additional work can begin. You can monitor the progress of files as they upload, and try again if the upload fails.")}</p>
+                <p>${_("Each video file that you upload needs to reach the edX video processing servers successfully before additional work can begin. You can monitor the progress of files as they upload, and try again if the upload fails.")}</p>
                 <h3 class="title-3">${_("Managing uploaded files")}</h3>
-                <p>${_("After a file uploads successfully, automated processing begins. After automated processing begins for a file it is listed under Previous Uploads as {em_start}In Progress{em_end}. When the status is {em_start}Complete{em_end}, edX assigns a unique video ID to the video file and you can add it to your course. If something goes wrong, the {em_start}Failed{em_end} status message appears. Check for problems in the file and upload a replacement.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("After a file uploads to edX successfully, automated processing, which takes {em_start}24 hours{em_end}, begins at edX and the file appears under Previous Uploads as {em_start}In Progress{em_end}. EdX assigns a unique video ID to each file as soon as possible so that you can add the videos to your course while other processing continues. After edX assigns a unique video ID to a file, its status on this page changes to  {em_start}Complete{em_end}. However, the external video host sites continue to process the files.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("{em_start}Important{em_end}: Processing by the external video host sites continues even after you see the {em_start}Complete{em_end} status on this page. It takes {em_start}24 hours{em_end} for your YouTube channel to list an uploaded video.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("The file statuses on this page refresh periodically. If something goes wrong, the file status changes to {em_start}Failed{em_end} status message appears. Check for problems in your original file and upload a replacement.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
There's something extra/missing in the "If something goes wrong..." sentence. I think you can delete "status message appears" since you added "...the file status changes to..."